years_to_hours.rb

```diff
@@ -0,0 +1,30 @@
+
+#Ruby Exercise: You will use the ruby development environment you set up on your machine for Backend 1. Write a program which asks the user for a number of years, and then prints out how many hours are in that many years. Then it asks for a number of decades, and prints out the number of minutes are in that many decades. Then it asks for the user's age, and prints out the number of seconds old the user is. Call this program years_to_hours.rb.
+
+#Here is the first part of the program:
+
+DAYS_IN_YEAR = 365
+HOURS_IN_DAY = 24
+MINUTES_IN_HOUR = 60
+SECONDS_IN_MINUTE = 60
+
+#Part 1: Ask the user for the number of years they would like to input
+puts "Enter a number of years"
+years = gets.chomp # this returns a string
```

Comment:
I believe you can string the `to_i` method onto the end of this line here.  Something like:
```
years = gets.chomp.to_i
```